### ISSUE TITLE
Migrating from 0.96 to 1.0

### ISSUE THREAD

**Issue by BackOrder** [doitsujin/dxvk#940]
I am migrating my games from 0.96 to 1.0 but the new script reports **File not found in wine prefix**. Is there anything wrong?

```
WINEPREFIX=/home/ian/Games/Steam/steamapps/compatdata/645630/pfx /home/ian/Workspace/dxvk-1.0/setup_dxvk.sh install
/home/ian/Games/Steam/steamapps/compatdata/645630/pfx/dosdevices/c:/windows/system32/dxgi.dll: File not found in wine prefix

WINEPREFIX=/home/ian/Games/Steam/steamapps/compatdata/645630/pfx /home/ian/Workspace/dxvk-1.0/setup_dxvk.sh install --without-dxgi
/home/ian/Games/Steam/steamapps/compatdata/645630/pfx/dosdevices/c:/windows/system32/d3d10.dll: File not found in wine prefix
```

**Comment by doitsujin** [comment]
This happens when a previous dxvk install deleted the original fake-dlls created by wine. The new script no longer does this and instead creates a backup, but for that reason it will also fail if the files don't exist.
 
Run `wineboot -u` on your wine prefix once, after that, it should work fine.

**Comment by BackOrder** [comment]
> Run `wineboot -u` on your wine prefix once, after that, it should work fine.

This fixes the issue. There is also a noticeable performance improvement coming with 1.0. Thank you for your amazing work.

> **Edit**: I see you're using Proton, in that case you should put the files into `steamapps/common/Proton 3.16 Beta/dist/lib64/wine/dxvk` (x64) and `...../dist/lib/wine/dxvk` (x32), rather than changing individual wine prefixes. The setup script is not intended to be used with Proton.

Will it be necessary to remove the previous DXVK installation in those individual wine prefixes?

**Comment by doitsujin** [comment]
> Will it be necessary to remove the previous DXVK installation in those individual wine prefixes?

Probably, yes.

**Comment by BackOrder** [comment]
Alright, will do. Thank you.

**Comment by BackOrder** [comment]
DXVK_HUD reports that the previous version is still in effect. What am I doing wrong?

![20190227220820_1](https://user-images.githubusercontent.com/8844578/53538723-df97e280-3adc-11e9-8606-3e28daa2a239.jpg)

**Comment by Joshua-Ashton** [comment]
You should uninstall using the script/method you installed with.
If that was the wineverb you may be out of luck without some manual bodging.

It may be easier to create a new wine prefix.

**Comment by doitsujin** [comment]
> * Installed DXVK to Proton 3.16-7 Beta default prefix.

You're **not** supposed to install it into the default prefix, but to the path that I mentioned above. That's the directory where all the wine libraries etc. are stored.

**Comment by BackOrder** [comment]
My apologies for giving you so much troubles. I had followed your instructions but it was not working, hence turning to alternative solutions. It turns out that my Proton installation might have been the problem. I deleted the directory, forced Steam to reinstall it and then installed DXVK 1.0 according to your instructions. This however works out perfectly.

Thank you for your time. Very much appreciated.
